migrace/021.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+// Vytvoři tabulyu pro uživatelské slevy
```

Comment:
překlep

Info: dost jsem přemýšlel, jestli je dobrý mít na to separátní tabulku nebo flag v tabulce platby. Nakonec, i vzhledem k tomu, že se to má chovat jako sleva a ne arbitrárně připsaná částka (viz dál), tak ta separátní tabulka asi je přece jen o něco lepší, takže ji nechme. Byť ta (nutná) duplikace s platbami není bůhví co.

Reply:
Godricu, jen pozor na název migračního skriptu... záleží na pořadí, jak budeš odbavovat... už je tam 21 u promlčení zůstatků a 22 u odstranění slev a 23 u odstranění Parconu.

Reply:
Mám to přejmenovat? Ještě budu mít v dalším PR určitě ještě jednu migraci.

Reply:
Abychom se nezabetonovali na pořadí, tak bych asi prozatím dělal tak, že se budem tvářit jakoby nic a dělat review bez ohledu na to, jaké čísla migračních skriptů jsou v PR. Jak vyřešíme zbytek reviews na daném PR, tak finálně vyzvu k přečíslování (pokud bude třeba) a pak to rovnou sloučím.

Aktuálně máme rozjetých hodně PR s paralelníma změnama databáze. Spíš počítám, že toto nebude běžný stav, takže pro teď budeme přečíslovávat a do budoucna se problém snad vyřeší sám. Pokud se ukáže, že tomu tak není, zkusíme to vymyslet nějaké lépe asi na poradě.